bin/improver-spot-extract

```diff
@@ -226,17 +221,14 @@ def main():
         with open(args.constants_path, 'r') as input_file:
             config_constants = json.load(input_file)
 
-    # Search directory structure for all files relevant to current diagnostic.
-    diagnostic_names = [diagnostics[diagnostic]["filepath"]
-                        for diagnostic in diagnostics.keys()]
-
     # Create a list of files that will need to be read.
-    all_available_files = glob.glob(args.data_path+"/*")
+    all_available_files = glob.glob(args.data_path)
```

Comment:
I've left this as it is, and modified the docstring of the command line argument to indicate that `args.data_path` is now expected as a file path i.e. `/path/to/file.nc`, rather than `/path/to/`. This fits with the current usage of SpotData, however, this may need to be revisited, once we're clearer on the usage of SpotData.